packages/devextreme-angular/gulpfile.js

```diff
@@ -192,6 +184,30 @@ gulp.task('clean.tests', () => {
   return del([outputFolderPath]);
 });
 
+gulp.task('add.package.json', (done) => {
```

Comment:
I am not sure that we need to do it. We need to test native solution of ng-packagr